NetKet/Operator/graph_operator.hpp

```diff
@@ -102,14 +103,12 @@ class GraphHamiltonian : public AbstractOperator {
     newconfs.clear();
     mel.resize(0);
 
-    for (std::size_t i = 0; i < operators_.size(); i++) {
-      operators_[i].AddConn(v, mel, connectors, newconfs);
-    }
+    operator_.AddConn(v, mel, connectors, newconfs);
```

Comment:
Can we use FindConn here instead of AddConn? Would that be a problem? I wanted to remove the method AddConn in Operator, since it seems unecessary in the new version!

Reply:
I removed it and it doesn't seem to cause any problems.